configs/wf_configs/cp_process_singlecells.yaml

```diff
@@ -42,6 +48,23 @@ cytotable_convert:
     preset: cellprofiler_sqlite
     log_level: ERROR
 
+annotate_configs:
+  params:
+    join_on:
+      - Metadata_well_position
+      - Image_Metadata_Well
+    add_metadata_id_to_platemap: True
+    format_broad_cmap: False
+    clean_cellprofiler: True
+    external_metadata: "none"
```

Comment:
I don't have a lot of experience with what the norm is for these `.yaml` files but is it possible to add hints to what the possibilities are for each param? ex. `dest_datatype: parquet` what are the possible input types? I am not sure is this is doable or a norm just a thought